specification.md

```diff
@@ -1634,6 +1640,69 @@ immediately after successful connection each Agent will acquire their own unique
 connection credentials. This way individual Agent's credentials may be revoked
 without disrupting the access to all other Agents.
 
+#### Agent-initiated CA trust Flow
+
+Status: [Development]
+
+This is an Agent-initiated flow that allows an agent to connect to a Server
+without trusting its CA. Initially the agent will connect without any form of
```

Comment:
It is not really "without trusting". The Agent will trust the CA once the flow is completed. It is just that the Agent does not have the CA certificate initially. Perhaps reword it like this?
```suggestion
without have the possession of the Server's CA initially. Initially the agent will connect without any form of
```